pkgs/build-support/trivial-builders.nix

```diff
@@ -132,13 +132,12 @@ rec {
     , destination ? ""   # relative path appended to $out eg "/bin/foo"
     , checkPhase ? ""    # syntax checks, e.g. for scripts
     , meta ? { }
+    , allowSubstitutes ? false
+    , preferLocalBuild ? true
```

Comment:
Those are unused.

Reply:
Oops, forgot to inherit these below. Fixed!